packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -167,6 +167,12 @@ const Mercury = WebexPlugin.extend({
 
     Promise.all([this._prepareUrl(socketUrl), this.webex.credentials.getUserToken()])
       .then(([webSocketUrl, token]) => {
+        if (!this.backoffCall) {
+          this.logger.info('mercury: prevent socket open when backOffCall no longer defined');
```

Comment:
```suggestion
          this.logger.info('mercury: prevent socket open when backoffCall no longer defined');
```
Capitalization didn't match the actual property.